### ISSUE TITLE
Send batch email with template

### ISSUE THREAD

**Issue by RaasAhsan** [pastjean/postmark-rs#26]
I am interested in sending batches of email with Postmark templates. This corresponds to the API here: https://postmarkapp.com/developer/api/templates-api#send-batch-with-templates . I see an API method to submit a batch of regular email, but can't seem to find one for batch templates. Is this possible today or does it need to be implemented?

Thanks for the great library!

**Comment by pastjean** [comment]
I haven't looked at postmark api in a while. We don't have the API call currently. If you submit a PR with that new call I'll be happy to put it in.

Otherwise it's pretty easy to implement a new call without a PR even. Just need to implement the Endpoint trait https://github.com/pastjean/postmark-rs/blob/main/src/client.rs#L10

**Comment by reneklacan** [comment]
Created [PR](https://github.com/pastjean/postmark-rs/pull/27) which implements this

@RaasAhsan in the meanwhile you can use it with:

```toml
postmark = { git = "https://github.com/reneklacan/postmark-rs", branch = "batch-with-templates" }
```

**Comment by pastjean** [comment]
Fixed in #27 and released in #28 (v0.10.2)

(: enjoy
